keras_cv/models/__internal__/unet.py

```diff
@@ -178,9 +178,7 @@ def UNet(
     function that acts on tensors as inputs.
 
     Reference:
-        - [U-Net: Convolutional Networks for Biomedical Image Segmentation]
```

Comment:
We cannot replace an arXiv link with a bit.ly link. To override the error of the line being too long, put `  # noqa: E501` at the end of the docstring